<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetAttributesDockWidgetBase</class>
 <widget class="QDockWidget" name="GetAttributesDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>get_attributes</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select a layer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Yes!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="showAttributes"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
